Cargo.toml

```diff
@@ -12,15 +12,15 @@ keywords = ["async", "specta", "rspc", "typescript", "typesafe"]
 categories = ["web-programming", "asynchronous"]
 
 [dependencies]
-specta = { version = "0.0.6", features = ["function", "tauri", "openapi"] }
-openapiv3 = "1.0.1"
-paste = "1.0.9"
+# specta = { version = "0.0.6", features = ["function", "tauri"] }
```

Comment:
Specta update needs to be released before this is released. Putting this all in a monorepo would make this easier.